espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/GetSessions.java

```diff
@@ -0,0 +1,26 @@
+package io.appium.espressoserver.lib.handlers;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+import io.appium.espressoserver.lib.handlers.exceptions.AppiumException;
+import io.appium.espressoserver.lib.model.AppiumParams;
+import io.appium.espressoserver.lib.model.Session;
+
+public class GetSessions implements RequestHandler<AppiumParams, Collection<String>> {
+
+    @Override
+    @Nullable
+    public Collection<String> handle(AppiumParams params) throws AppiumException {
+        // Returns all of
+        List<String> list = new ArrayList<>();
+        if(Session.getGlobalSession() != null)
+            list.add(Session.getGlobalSession().getId());
```

Comment:
return Collections.unmodifiableList(Collections.singletonList(Session.getGlobalSession().getId()));

I assume it makes sense to move such wrapper into base class, because it's going to be used more than one time